ExchangeActiveSync/Source/Common/Common/Enums.cs

```diff
@@ -23,7 +23,12 @@ public enum SutVersion
         /// <summary>
         /// The SUT is Exchange Server 2016
         /// </summary>
-        ExchangeServer2016
+        ExchangeServer2016,
+
+        /// <summary>
+        /// The SUT is Exchange Server 2016

```

Comment:
should be "The SUT is Exchange Server 2019"

Reply:
Just ExchangeServer2019 should be Exchange Server 2019? How about ExchangeServer2007, ExchangeServer2010, ExchangeServer2013 and ExchangeServer2016？